src/controllers/midi/midicontroller.cpp

```diff
@@ -537,3 +547,41 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
     qCWarning(m_logBase) << "MidiController: No script function specified for"
                          << MidiUtils::formatSysexMessage(getName(), data, timestamp);
 }
+
+void MidiController::makeInputHandler(const QJSValue& control, const QJSValue& scriptCode) {
```

Comment:
I think I'd prefer to use the values directly instead of packing them in a map because the map only results in extra overhead and it would allow this class to be simpler as we don't need to do the typechecking ourselves. Wdyt?
```suggestion
void MidiController::makeInputHandler(int status, int midino, const QJSValue& scriptCode) {
```